web/src/services/dataservice.js

```diff
@@ -11,19 +11,14 @@ var moment = require('moment');
 
 // API
 function protectedJsonRequest(endpoint, path, callback) {
-  let t = new Date().getTime();
-  let md = forge.md.sha256.create();
-  let s
-  if (endpoint.indexOf('localhost') != -1) {
-    s = md.update('development' + path + t).digest().toHex();
-  }
-  else {
-    s = md.update(ELECTRICITYMAP_PUBLIC_TOKEN + path + t).digest().toHex();
-  }
-  let req = d3.json(endpoint + path)
+  const now = new Date().getTime();
+  const md = forge.md.sha256.create();
+  const signature = md.update(ELECTRICITYMAP_PUBLIC_TOKEN + path + now).digest().toHex();
```

Comment:
Here we should use the "development" token in dev mode

Reply:
I know, you've already pass "development" token here https://github.com/tmrowco/electricitymap-contrib/blob/master/web/webpack.config.js#L46

`